src/jdk.jpackage/linux/classes/jdk/jpackage/internal/LinuxDebBundler.java

```diff
@@ -508,7 +509,8 @@ public String getID() {
 
     @Override
     public boolean supported(boolean runtimeInstaller) {
-        return Platform.isLinux() && (new ToolValidator(TOOL_DPKG_DEB).validate() == null);
+        return (OperatingSystem.isLinux() | OperatingSystem.isAix())
```

Comment:
Why use `|`? I think from the code logic, `||` is better.

```suggestion
        return (OperatingSystem.isLinux() || OperatingSystem.isAix())
```